docs/resources/Guild_Event.md

```diff
@@ -0,0 +1,101 @@
+| entity_id            | ?snowflake                                                                                                 | entity id                                                                                 |
+| entity_metadata      | [entity metadata](#DOCS_RESOURCES_GUILD_EVENT/guild-event-object-guild-event-entity-metadata)              | metadata for the event                                                                    |
+| sku_ids              | array of snowflakes                                                                                        | sku ids                                                                                   |
+| skus                 | array                                                                                                      | skus                                                                                      |
+| user_count?          | integer                                                                                                    | users subscribed to the event                                                             |
+
+###### Guild Scheduled Event Entity Types
+
+| Type           | Value |
+| -------------- | ----- |
+| NONE           | 0     |
+| STAGE_INSTANCE | 1     |
+| VOICE          | 2     |
+| EXTERNAL       | 3     |
+
+###### Guild Scheduled Event Status
+
+| Type      | Value |
+| --------- | ----- |
+| SCHEDULED | 1     |
+| ACTIVE    | 2     |
+| COMPLETED | 3     |
+| CANCELED  | 4     |
+
+###### Guild Event Entity Metadata
+
+| Field        | Type                | Description                       |
+| ------------ | ------------------- | --------------------------------- |
+| speaker_ids? | array of snowflakes | the speakers of the stage channel |
+
+## List Guild Events for Guild % GET /guilds/{guild.id#DOCS_RESOURCES_GUILD/guild-object}/events
+
+Returns a list of [Guild Events](#DOCS_RESOURCES_GUILD_EVENT/guild-event-object) in the guild.
+
+###### Query String Params
+
+| Field            | Type    | Description                   |
+| ---------------- | ------- | ----------------------------- |
+| with_user_count? | boolean | users subscribed to the event |
+
+## Create Guild Event % POST /guilds/{guild.id#DOCS_RESOURCES_GUILD/guild-object}/events
+
+Create a Guild Event in the guild. Returns a [Guild Event](#DOCS_RESOURCES_GUILD_EVENT/guild-event-object) object on success.
+
+###### JSON Params
+
+| Field                | Type                                                                                                       | Description                            |
+| -------------------- | ---------------------------------------------------------------------------------------------------------- | -------------------------------------- |
+| channel_id?          | snowflake                                                                                                  | the stage channel id of the event      |
```

Comment:
```suggestion
| channel_id?          | snowflake                                                                                                  | the channel id of the event            |
```

Reply:
so, it's channel id everywhere now except here, seems like you forgot to commit this one 👀